content/millennium/r4/conformance/structure-definition.md

```diff
@@ -96,6 +96,7 @@ Authorization is not required.
  [`formatted-text`]                                          | Link to the formatted text for a note. It is used in cases when the note is not a plain text.
  [`patient-friendly-display`]                                | An expression of the original string in terminology that patients should be able to understand.
  [`payment-collection-status`]                               | The status of the payment collection for the encounter.
+ [`pharmacy-verification-status`]                            | Represents whether a MedicationRequest has been verified by a pharmacist. Supported values are Does not need pharmacy verification, Needs pharmacy verification or Rejected by pharmacy.
```

Comment:
The rest of the definitions don't seem to indicate the supported values. We might be able to just leave that as part of the Structure definition itself, as well as the MedicationRequest page.

Reply:
Removed [commit](https://github.com/cerner/fhir.cerner.com/pull/609/commits/895786e1d26c542a51ad48e9bc22074506b9aac6)